http/test/workflow.test.ts

```diff
@@ -0,0 +1,67 @@
+import { Connection, WorkflowClient } from '@temporalio/client';
+import { Worker, DefaultLogger } from '@temporalio/worker';
+import { describe, before, after, afterEach, it } from 'mocha';
+import { example } from '../lib/workflows';
```

Comment:
Can these duplications of the outDir path be removed?

I would prefer that outDir only needed to be changed in one place: tsconfig.json

Reply:
This is unnecessary, you can import directly from `./workflows`

Reply:
I think this is a consequence of using ts-mocha to run the tests from the test folder, while the code under test is precompiled into lib.